src/calculateRentalCost.js

```diff
@@ -5,6 +5,25 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let costOfrent = 40 * days;
```

Comment:
Consider using `const` instead of `let` for `costOfrent` since its value is not being reassigned, but rather a new value is returned when discounts are applied. This adheres to checklist item 3, preferring `const` over `let` where possible to avoid unintentional changes.